.yarn/versions/a07c2b66.yml

```diff
@@ -0,0 +1,13 @@
+releases:
+  "@radix-ui/react-context-menu": patch
+  "@radix-ui/react-dropdown-menu": patch
+  "@radix-ui/react-menu": patch
+  "@radix-ui/react-radio-group": patch
+  "@radix-ui/react-roving-focus": patch
+  "@radix-ui/react-tabs": patch
+  "@radix-ui/react-toggle-group": patch
+  "@radix-ui/react-toolbar": patch
+
+declined:
+  - primitives
+  - "@radix-ui/react-menubar"
```

Comment:
Do I have these right?

Reply:
`"@radix-ui/react-menubar"` should be major right?

Reply:
Good question, should the first release be a straight major?